src/ParsedText.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import ReactNative from 'react-native';
```

Comment:
Are there any significant changes in this file that need to be reviewed? Most of them look like Formatting changes.

Reply:
The default export of `react-native` has been removed from web and will be removed from native soon. I opened this PR right after deleting it. It was removed because it causes all of the RN module to be bundled regardless of if you are using them or not.